web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -152,6 +163,23 @@ function joinup_core_form_rdf_entity_form_alter(&$form, FormStateInterface $form
 
 }
 
+/**
+ * Displays status message after the user saved the related RDF content.
+ */
+function _joinup_core_message_handler(&$form, FormStateInterface $form_state) {
+  /** @var \Drupal\rdf_entity\Form\RdfForm $form_object */
+  $form_object = $form_state->getFormObject();
+  /** @var \Drupal\rdf_entity\RdfInterface $rdf */
+  $rdf = $form_object->getEntity();
+
+  /** @var \Drupal\Core\Messenger\MessengerInterface $messenger */
+  $messenger = \Drupal::service('messenger');
+
+  // Add a status message when it was saved.
+  $messenger->addStatus(t('Thank you for proposing a @entity_label. 
```

Comment:
Let's not wrap the message in this way. Better on a single line.

Reply:
Changed!